products/compute/api.yaml

```diff
@@ -659,7 +658,85 @@ objects:
           that was later deleted and recreated under the same name, the source
           image ID would identify the exact version of the image that was used.
         output: true
-<%= indent(compile('products/compute/disk_parameters.yaml'), 6) %>
```

Comment:
Want to inline this into `RegionDisk` too and delete it?

Reply:
I'm happy to do that in a separate PR. Pulling it into this PR is scope creep that's not really required.

Reply:
I see where you're coming from, but this PR is a 3 line change otherwise; I think it's a perfectly reasonable PR to inline the file in both places and then making this minor change as a separate commit. You're also correct that splitting it into two distinct PRs would be even better imo.

Reply:
I'll follow up with the rest of the refactors in another PR 👍